src/ophyd_async/epics/adpilatus/_pilatus_io.py

```diff
@@ -14,11 +16,7 @@ class PilatusTriggerMode(StrictEnum):
 
 
 class PilatusDriverIO(adcore.ADBaseIO):
-    """Mirrors the interface provided by ADPilatus/db/pilatus.template."""
-
-    def __init__(self, prefix: str, name: str = "") -> None:
-        self.trigger_mode = epics_signal_rw_rbv(
-            PilatusTriggerMode, prefix + "TriggerMode"
-        )
-        self.armed = epics_signal_r(bool, prefix + "Armed")
-        super().__init__(prefix, name=name)
+    """This mirrors the interface provided by ADPilatus/db/pilatus.template."""
+    """See HTML docs at https://areadetector.github.io/areaDetector/ADPilatus/pilatusDoc.html"""
```

Comment:
Sorry, last thing! Could we add a brief description here such as:
```suggestion
    """Driver for the Pilatus pixel array detectors"""
    
    """This mirrors the interface provided by ADPilatus/db/pilatus.template."""
    """See HTML docs at https://areadetector.github.io/areaDetector/ADPilatus/pilatusDoc.html"""
```